dependencies.yaml

```
@@ -377,6 +378,33 @@ dependencies:
               - *libcusolver114
               - *libcusparse_dev114
               - *libcusparse114
+  cuda_wheels:
+    specific:
+      - output_types: pyproject
+        matrices:
+          - matrix:
+              cuda: "12.*"
+              cuda_suffixed: "true"
+              use_cuda_wheels: "true"
+            packages:
```

```suggestion
            packages: &cuda_wheels_cu12
```

For re-use in the fallback matrix.